opengcs/.travis.yml

```diff
@@ -1,8 +0,0 @@
-services:
```

Comment:
Are we adding gcs building to the main CI?

Reply:
Yea that was the plan. Probably be better to do that in here as well instead of a separate pr

Reply:
I think addressing this here would be best.